releasenotes/notes/add-fit-model-and-remove-curve_fitter-arg-4a0effb5f9b88ba9.yaml

```diff
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    :attr:`CurveAnalysis.fit_model` and :attr:`CurveAnalysis.parameters` have been
+    added to the curve analysis and its subclasses. These properties return
+    :class:`FitModel` instance that represents a fit function of the analysis
+    and parameters to be fit with the analysis, respectively.
+  - |
+    New classes :class:`SingleFitFunction` and :class:`CompositeFitFunction` have been
+    added to the curve analysis module. These classes abstract the curve definition
+    hard-coded in the class attribute :attr:`CurveAnalysis.__series__` as a
+    Python function-like object. This may help user to grasp the fitting model
+    without reading the code.
```

Comment:
I don't understand this paragraph (except for the first sentence). Please rephrase it.